components/base-war/src/main/webapp/resources/js/dragdrop.js

```diff
@@ -0,0 +1,974 @@
+// script.aculo.us dragdrop.js v1.9.0, Thu Dec 23 16:54:48 -0500 2010
```

Comment:
Also, make a pull request against [this file](https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/scriptaculous/dragdrop.js).